tools/repl/src/repl.rs

```diff
@@ -223,14 +241,14 @@ impl REPL {
 fn print_help() {
     println!(
         "Commands:\n\n\
-            n/new [config_path]                       create a new service (current will be removed)\n\
-            l/load <module_name> <module_path>        load a new Wasm module\n\
-            u/unload <module_name>                    unload a Wasm module\n\
-            c/call <module_name> <func_name> [args]   call function with given name from given module\n\
-            i/interface                               print public interface of all loaded modules\n\
-            e/envs <module_name>                      print environment variables of a module\n\
-            f/fs <module_name>                        print filesystem state of a module\n\
-            h/help                                    print this message\n\
-            q/quit/Ctrl-C                             exit"
+            n/new [config_path]                                   create a new service (current will be removed)\n\
+            l/load <module_name> <module_path>                    load a new Wasm module\n\
+            u/unload <module_name>                                unload a Wasm module\n\
+            c/call <module_name> <func_name> <args> [call_params] call function with given name from given module\n\
```

Comment:
```suggestion
            c/call <module_name> <func_name> [args] [call_params] call function with given name from given module\n\
```
Or args now passed without brackets?

Reply:
I interpreted notation`<name>` as "argument name is required", and `[name]` as "argument name is optional", args were required before this PR